README.md

```diff
@@ -3,11 +3,15 @@ Some shell scripts related to Thermo Fisher raw format conversion using msconver
 
 ## docker scripts
 ```shell
-docker pull chambm/pwiz-skyline-i-agree-to-the-vendor-licenses
-docker run -itd -v <local directory>:<container directory> <image ID> /bin/bash
-docker exec -it <container names> bash
+> docker build -t Nesvilab/msconvert-scripts .
```

Comment:
Why creating another docker image since there is already a docker image `chambm/pwiz-skyline-i-agree-to-the-vendor-licenses`? Seems a little redundant and cumbersome. 

Thanks,

Fengchao

Reply:
From the instruction, I think it's relatively hard to use and also without installing parallel package, it won't work directly through the original image.

Reply:
The old and new way require the same number of commands, which is three, to get the container running, but the new way requires an additional Dockerfile and a new image. I am sorry that I can't see why it is easier than the old way.

`parallel` is optional. Without parallel, just use this script https://github.com/Nesvilab/msconvert-scripts/blob/main/run_msconvert.sh

If wanting to make the process easier, I think you should add the `RUN apt-get update && apt-get install -yq parallel` to the `chambm/pwiz-skyline-i-agree-to-the-vendor-licenses` and send a pull request to their repository. It will save the parallel dependency problem once-for-all.

Best,

Fengchao